compiler+runtime/src/cpp/jank/runtime/core/seq.cpp

```diff
@@ -57,6 +57,17 @@ namespace jank::runtime
       o);
   }
 
+  native_bool is_seqable(object_ptr const o)
+  {
+    return visit_object(
+      [=](auto const typed_o) -> native_bool {
+        using T = typename decltype(typed_o)::value_type;
+
+        return behavior::seqable<T>;
+      },
+      o);
```

Comment:
Oh, we have `visit_seqable`, which takes two fns. The first can just return true and the second can just return false.